static/app/components/replays/keyMetrics.tsx

```diff
@@ -9,38 +9,24 @@ import space from 'sentry/styles/space';
  */
 
 type Props = {
-  keyName: React.ReactNode;
-  value: React.ReactNode;
+  children?: React.ReactChild;
 };
 
-export const KeyMetrics = styled('dl')`
-  margin: 0; /* Reset default dl styles */
-
-  display: grid;
-  grid-template-rows: auto auto;
-  grid-auto-columns: 1fr;
-  grid-auto-flow: column;
-  gap: ${space(0.5)} ${space(2)};
-
-  text-align: right;
+export const KeyMetrics = styled('div')`
+  display: flex;
+  gap: ${space(3)};
+  align-items: center;
+  font-size: ${p => p.theme.fontSizeMedium};
 `;
 
-export const KeyMetricData = ({keyName, value}: Props) => {
-  return (
-    <Fragment>
-      <Key>{keyName}</Key>
-      <Value>{value}</Value>
-    </Fragment>
-  );
+export const KeyMetricData = ({children}: Props) => {
+  return <Value>{children}</Value>;
 };
 
-const Key = styled('dt')`
-  color: ${p => p.theme.subText};
-  font-size: 14px;
-  font-weight: bold;
-`;
-
-const Value = styled('dt')`
+const Value = styled('div')`
```

Comment:
looks like there's an extra react component which we probably don't need.

- would it work if we remove the existing `KeyMetricData` which isn't doing much
then we could rename `const Value =` to be anything you like, and export it

Reply:
I deleted the component with its storybook as I don't see them necessary anymore and moved the styled components into `detailLayout.tsx` file